lms/templates/fields/field_textarea.underscore

```diff
@@ -8,11 +8,19 @@
     </div>
 
     <div class="u-field-value" id="u-field-value-<%- id %>" aria-labelledby="u-field-title-<%- id %>"><%
-        if (mode === 'edit') {
-            %><textarea id="u-field-textarea-<%- id %>" rows="4" aria-describedby="u-field-placeholder-value-<%- id %>"><%- value %></textarea><%
-        } else {
+        if (mode === 'edit') {%>
+            <% if (descriptionMessage) { %>
+                <textarea id="u-field-textarea-<%- id %>" rows="4" aria-describedby="u-field-description-help-<%- id %>"><%- value %></textarea>
+            <% } else { %>
+                <textarea id="u-field-textarea-<%- id %>" rows="4" aria-describedby="u-field-placeholder-value-<%- id %>"><%- value %></textarea>
+            <% } %>
+        <% } else {
```

Comment:
This will make the intention of the code a bit more clearer:
```
        var textareaDescribedByPrefix = (message ? 'u-field-message-help-' : 'u-field-placeholder-value-');
        if (mode === 'edit') {
            %><textarea id="u-field-textarea-<%- id %>" rows="4" aria-describedby="<%- textareaDescribedByPrefix %>-<%- id %>"><%- value %></textarea><%
        } else {
```